src/model/InventoryModel.ts

```diff
@@ -1,4 +1,6 @@
 export type Inventory = {
```

Comment:
your inventory can host multiple objects, so it will take a property which will contain  Object[ ] type